components/org.wso2.carbon.identity.api.user.common/src/main/java/org/wso2/carbon/identity/api/user/common/Util.java

```diff
@@ -82,4 +90,54 @@ public static String resolveUserIdFromUser(User user) {
                     .build(log, e, "Invalid userId: " + user.getUserName()));
         }
     }
+
+    /**
+     * Validate whether the given id is valid user id in the user store or in the session management data stores.
+     *
+     * @param realmService  realm service
+     * @param userId    unique user id of the user
+     * @param tenantDomain  tenant domain of the user
+     */
+    public static void validateUserId(RealmService realmService, String userId, String tenantDomain) {
+
+        if (StringUtils.isEmpty(userId)) {
+            throw new WebApplicationException("UserID is empty.");
+        }
+        boolean isUserValid;
+
+        try {
+            isUserValid = UserSessionStore.getInstance().isExistingUser(userId);
+            if (!isUserValid) {
+                UniqueIDUserStoreManager uniqueIdEnabledUserStoreManager =
+                        getUniqueIdEnabledUserStoreManager(realmService, tenantDomain);
+                isUserValid = uniqueIdEnabledUserStoreManager.isExistingUserWithID(userId);
+            }
+        } catch (org.wso2.carbon.user.api.UserStoreException | UserSessionException e) {
```

Comment:
Can't u straightly call UserStoreException without the package names?

Reply:
updated